File: PRAK502-2210817120009-Najah_Maisyaroh.py
def hitungan(nilai1, nilai2):
    nilai1=int(nilai1); nilai2=int(nilai2)
    return nilai1 - nilai2
def mutlak(angka):
    angka=int(angka)
    return abs(angka)
a,c,b,d=map(int, input().split())
Hasil=hitungan(a, b) + hitungan(c, d)
print(mutlak(Hasil))

File: PRAK502-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
#include <math.h>
int Hasil;
int hitung(int nilai1, int nilai2){
    Hasil = nilai1 - nilai2;}
int mutlak(int angka){
    if (angka<0){
        angka=-(angka);}
    return (angka);}
int main(){
    int a,b,c,d, Hasil;
    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);
    Hasil = hitung(a,b) + hitung(c,d);
    printf("%d", mutlak(Hasil));
    return 0;
}